### ISSUE TITLE
Simplify Project Setup

### ISSUE THREAD

**Issue by dpoetzsch** [yiotro/Antiyoy#42]
Hi,

as said in #13, I want to help improve antiyoy.

I managed to set the game up and get a working config (actually the houses and units are not shown on the map, i guess there is still some wrong configuration somewhere).

To be really honeest, setting the whole thing up was a huge pain. Your guide helped a lot but I still had to figure out some stuff by myself. In core I had to comment out some android specific stuff (in `GetAndroidClipboardContents` and `LanguagesManager`) + set the language level of core to 1.7 as well.

I wondered, is there any specific reason why you don't keep the whole project in git? I figure that would make development much much easier and the results would be comparable accross devices (no unshared configs that might lead to different results on different machines).

If this is appreciated, I'd be happy to give this a go!

**Comment by yiotro** [comment]
About those black textures: looks like I forgot to update assets in this repo after last update. I'll do it on Monday or Tuesday.

About commenting out code: I'm not an expert in this, but I suspect that it can be fixed by changing how libs are linked in your project, but that's just my guess.

About why I don't just keep whole project here: I'm not sure that it's actually safe. What if some files will contain some valuable info?

**Comment by dpoetzsch** [comment]
Well, those textures are not exactly black, just missing; see here: [screenshot_20180211-005157](https://user-images.githubusercontent.com/7511464/36067809-f1799294-0ec5-11e8-849a-3a7fcb1e0e4c.png)

I'll look into that linking. It'd be interesting how your setup differs from mine. Also, maybe there is a clean way of separating android stuff from core which would probably be a good step towards #22.

Actually this is exactly the reason why I suggest to put the project setup into the repository. Obviously your setup is already different from mine.

I'm not sure what valuable info you are referring to. Of course one would put all sensitive data like passwords or keys into files that are ignored using `.gitignore` and not share those. There is standard ignore lists for android projects, probably also for libgdx projects. This is a pretty standard technique for android projects, if you have a look at other android projects like for example [PocketHub](https://github.com/pockethub/PocketHub). What other valuable info are you thinking of?

**Comment by yiotro** [comment]
Update about missing textures: I've checked it out and it looks like assets were ok. That's strange...

I know about gitignore, but I still don't want to use it because I can't be sure that I'll hide everything valuable there.

**Comment by dpoetzsch** [comment]
My workflow usually is as follows: Before I do a commit I'll do a `git diff` and review and potentially clean up all my changes. That way I have full control of what gets committed and pushed to public.

For a basis, in the setup as it is now, there cannot be valuable information of yours as **I** generated the whole setup. From now on, it suffices to review all changes. Isn't a manual copy of changed files to the repository similar to this review?

**Comment by yiotro** [comment]
I don't think that it's really needed. As for now this repo already has everything that you need to recreate antiyoy libgdx project.

**Comment by dpoetzsch** [comment]
Well, it's your repository ;-)

However, then I think the instructions need to be updated at least because by just following them I cannot recreate the project ;-)

Also, am I correct that if I want to submit a PR, I'll have to copy the files from my project setup manually to the forked repository or do you have a different workflow?

**Comment by yiotro** [comment]
> Also, am I correct that if I want to submit a PR, I'll have to copy the files from my project setup manually to the forked repository or do you have a different workflow?

Probably yes. But I have to say that I prefer to work on this project on my own.
